concourse/scripts/kick-off-pipeline.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Required variables are:
+# - DEPLOY_ENV
+# - AWS_ACCOUNT
+#
+# Optional variables:
+# - PIPELINES_TO_UPDATE
+
+set -u
+set -e
+
+if [ ! -d "./paas-cf" ]; then
+  echo "Resource paas-cf must be checkout"
+  exit 1
+fi
+
+if [ "${ENABLE_MORNING_DEPLOYMENT}" != "true" ]; then
```

Comment:
Again, do we need this conditional in here given the pipeline that calls this will only be deployed when this flag is enabled?